components/table/TableHead.js

```diff
@@ -5,7 +5,8 @@ const factory = (Checkbox) => {
     let selectCell;
     const contentCells = Object.keys(model).map((key) => {
       const name = model[key].title || key;
-      return <th key={key}>{name}</th>;
+      const width = model[key].width;
+      return <th key={key} style={{width}}>{name}</th>;
```

Comment:
I know it's coming from the model, but I mean that it could be more general if you just use `style` instead of with so the model defines a full style object. This way you could customize the whole header cell not just width.